src/views/Transitioner.js

```diff
@@ -105,24 +111,57 @@ class Transitioner extends React.Component<Props, State> {
       this.props.navigation.state
     );
 
-    if (nextScenes === this.state.scenes) {
+    const noSceneChange = nextScenes === this.state.scenes;
```

Comment:
We probably don't need any of this logic if we aren't doing the update redux without animation.  We had that problem where it froze hence we went the purge route.

That is pretty much everything in this file before line 215 `if (this.props.isFlipTransition) {` is probably not required.

Might be cleaner to not have it all if it's not being used.

Reply:
@sdg9 done